onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -7,6 +7,9 @@
         "^test_batchnorm_epsilon_training_mode",
         "^test_batchnorm_example_old",
         "^test_batchnorm_example_training_mode",
+        "^convtranspose_autopad_same",
```

Comment:
i think this needs to be "^test_convtranspose_autopad_same"

Reply:
I did that one too